be/src/vec/data_types/serde/data_type_datetimev2_serde.cpp

```diff
@@ -79,8 +79,8 @@ Status DataTypeDateTimeV2SerDe::deserialize_one_cell_from_json(IColumn& column,
         }
 
     } else if (ReadBuffer rb(slice.data, slice.size);
-               !read_datetime_v2_text_impl<UInt64>(val, rb)) {
-        return Status::InvalidDataFormat("parse date fail, string: '{}'",
+               !read_datetime_v2_text_impl<UInt64>(val, rb, scale)) {
+        return Status::InvalidArgument("parse date fail, string: '{}'",
                                          std::string(rb.position(), rb.count()).c_str());
```

Comment:
why not pick related regression tests?

Reply:
the out file is generate from master which round behavior is not compatible for 2.0